blocklyc.html

```diff
@@ -97,329 +93,96 @@
         <script type="text/javascript" src="src/blockly/generators/propc/sensors.js"></script>
         <script type="text/javascript" src="src/blockly/generators/propc/heb.js"></script>
         <script type="text/javascript" src="src/blockly/generators/propc/s3.js"></script>
-
+        
         <!-- Style sheets -->
         <link href="src/lib/chartist.min.css" rel="stylesheet" type="text/css" />
         <link href="src/lib/bootstrap/core/css/bootstrap.min.css" rel="stylesheet" type="text/css" />
         <link href="src/style-editor.css" rel="stylesheet" type="text/css" />
         <link href="src/style-clientdownload.css" rel="stylesheet" type="text/css" />
+
+        <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Ubuntu|Rambla|Basic|Oxygen|Rubik|Nunito+Sans|Varela+Round|Puritan|Signika|Magra|Archivo|Itim|Neucha|Lato|Share+Tech|PT+Sans|Heebo|Cabin|ABeeZee|Arimo|Encode+Sans+Semi+Condensed|Exo+2|Open+Sans">
+        
+        
+        
+        
     </head>
+    
     <body>
         <div id="editor">
             <table id="content_table">
                 <tr>
                     <td>
-                        <nav class="navbar navbar-default clearfix" role="navigation">
+                        <nav class="navbar navbar-default clearfix">
                             <div style="width:100%;">
                                 <div style="display:inline;">
-                                    <span><a id="nav-logo" class="url-prefix" href="/">BlocklyProp<br><strong>Local</strong></a></span>
+                                    <a id="nav-logo" href="" class="url-prefix">BlocklyProp<br><strong>Developer</strong></a>
                                 </div>
                                 <div style="display:inline;">
-                                    <!-- BlocklyProp Client detection -->
-                                    <div style="width:100%; ">
-                                        <!-- authenticatd -->
-                                        <div style="display:inline; padding-left: 10px; line-height: 30px;" class="auth-true">
-                                            <!-- Looking for client connection -->
+                                    <div style="width:100%;">
+                                        <div class="auth-true" style="display: inline; padding-left: 10px; line-height: 30px;">
                                             <span id="client-searching" class="bp-client-warning">
-                                                <a class="client-searching-link" href="#" data-toggle="modal" data-target="#client-download-modal">
```

Comment:
Are the svg images getting replaced with something from the bootstrap library?

Reply:
SVG images will be sourced from editor.js.  Instead, the HTML will just contain placeholders.